src/app/(private)/(routes)/admin/courses/components/columns.tsx

```diff
@@ -43,12 +47,26 @@ export const columns: ColumnDef<CourseColumn>[] = [
       </Badge>
     )
   },
-
   {
     accessorKey: 'isPaid',
     header: 'Paid',
     cell: ({ row }) => <span>{row.original.isPaid ? 'Paid' : 'Free'}</span>
   },
+  {
+    accessorKey: 'isPending',
+    header: ({ column }) => {
+      return (
+        <Button
+          variant="ghost"
+          onClick={() => column.toggleSorting(column.getIsSorted() === 'asc')}
+        >
+          Pending
+          <ArrowUpDown className="ml-2 h-4 w-4" />
+        </Button>
+      )
+    },
+    cell: ({ row }) => <span>{row.original.isPending ? 'Yes' : 'No'}</span>
```

Comment:
Aqui é apenas capricho visual. Mas se puder, ao inves desse span cria um componente Badge (Semelhante a coluna de level do curso) e coloca uma cor identificadora. Por exemplo, quando isPending for true vc coloca a cor laranja no badge e o texto "Pending". Quando for false coloca a cor verde e o texto "Approved"